package.json

```diff
@@ -8,7 +8,11 @@
     "@heroku-cli/command": "^8.1.23",
     "@oclif/command": "^1",
     "@oclif/config": "^1",
+    "@types/debug": "^0.0.30",
+    "@types/shell-escape": "^0.2.0",
+    "debug": "^3.1.0",
     "node-fetch": "^2.1.2",
+    "shell-escape": "^0.2.0",
```

Comment:
why is there `shell-escape` and `@types/shell-escape` (same with debug)?

Reply:
you need `@types` for TypeScript typing information.

Reply:
not a major thing, but ideally the `@types/*` packages should all go into `devDependencies` so users don't have to download them to run the plugin